addons/mass_mailing/tests/test_mailing_retry.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.mass_mailing.tests.common import MassMailCommon
+from odoo.addons.base.tests.test_ir_cron import CronMixinCase
+from odoo.tests.common import users
+
+class TestMailingRetry(MassMailCommon, CronMixinCase):
+
+    @users('user_marketing')
+    def test_mailing_retry_immediate_trigger(self):
+        default_list_ids = self.env['mailing.list'].with_context(self._test_context).create({
```

Comment:
Could we instead use the helper method "_create_mailing_list" and use "self.mailing_list_1" here below?

That way we re-use a bit the helper data instead of creating new ones.